docs/core/tutorials/using-mstest-on-windows.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Use MSTest with .NET Core on Windows
+description: How to use MSTest with .NET Core on Windows, using Visual Studio 2015
+keywords: MSTest, .NET, .NET Core
+author: 
+manager: 
+ms.date: 08/18/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 
+---
+
+# Unit testing with MSTest and .NET Core on Windows, using Visual Studio 2015
+
+by [Nicolò Caradini](https://github.com/ncarandini)
+
+While xUnit could be a better choice when targeting multiple platforms, with .NET Core on Windows is also possible to use MSUnit.
+
+## Prerequisites
+
+Follow the instructions on [Getting started with .NET Core on Windows](./using-mstest-on-windows.md) to create the library project.
+
+### Writing the test project using MSTest
+
+1. In Solution Explorer, open the context menu for the **Solution** node and choose **Add**, **New Solution Folder**. Name the folder "test". 
+   This is only a solution folder, not a physical folder.
+
+2. Open the context menu for the **test** folder and choose **Add**. **New Project**. In the **New Project** dialog, choose **Console Application (.NET Core)**. Name it "TestLibrary" and explicitly put it under the `Golden\test` path. 
+
+   > **Important**
+   >
+   > The project needs to be a console application, not a class library.
```

Comment:
This should be formatted as (per [the docs template](https://github.com/dotnet/core-docs/blob/master/styleguide/template.md#alerts)):

```md
> [!IMPORTANT]
> The project needs to be a console application, not a class library.
```